### DISCUSSION TITLE
https://www.facebook.com/profile.php?id=100040677634079

### DISCUSSION THREAD

**Discussion by 0963166991** [community/community#9422] [Other Feature Feedback, Questions, & Ideas]
It's hacked. Can you save me? I do not have an email address. Can you help me?

**Comment by 0963166991** [comment]
https://www.facebook.com/profile.php?id=100040677634079

**Comment by 0963166991** [comment]
It's hacked. Can you save me? I do not have an email address. Can you help me?

**Comment by KlausEverWalkingDev** [comment]
This is not a Facebook support forum. Maybe you can get help there: https://www.facebook.com/help/1126628984024935
